authbridge/demos/finance-sparc/scripts/setup_keycloak_finance.py

```
@@ -0,0 +1,97 @@
+            "name": scope_name, "protocol": "openid-connect",
+            "attributes": {"include.in.token.scope": "true", "display.on.consent.screen": "false"},
+        }, skip_exists=True)
+    print(f"  client scope {scope_name} -> {scope_id}")
+    try:
+        kc.add_mapper_to_client_scope(scope_id, {
+            "name": scope_name + "-aud-mapper", "protocol": "openid-connect",
+            "protocolMapper": "oidc-audience-mapper",
+            "config": {"included.custom.audience": agent_spiffe,
+                       "id.token.claim": "false", "access.token.claim": "true"},
+        })
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)